Develop

DmJSCourse opened the pull request:
https://dmjscourse.github.io/Thrive_talk/

vsm2k17 commented:
https://www.loom.com/share/78fb2c720d6a4f1a93a7d2417462f24c

vsm2k17 commented:
https://www.notion.so/ThriveTalk-review-for-Lukianets-Dmytro-ae43ead43497494d9f17e5262aa57923

irenhh commented:
![image](https://user-images.githubusercontent.com/33497431/88520123-d37ac200-cffb-11ea-9f62-0cd2bc174dbe.png)
seems like something is broken, check the design for the first screen

=====

<img width="103" alt="image" src="https://user-images.githubusercontent.com/33497431/88520222-f1e0bd80-cffb-11ea-8ecc-32ceb4093b37.png">
fix underline, it needs to have the same width as the content + transition is too slow

=====

<img width="1272" alt="image" src="https://user-images.githubusercontent.com/33497431/88520407-3409ff00-cffc-11ea-84ac-d5c4dc38f435.png">
indents (check the scree size on the screenshot)

=====

<img width="287" alt="image" src="https://user-images.githubusercontent.com/33497431/88520503-526ffa80-cffc-11ea-9f52-6203c896c9a5.png">
text should be aligned to the left of the decorative line

=====

1280px is still a desktop size, fix your page for this size 

=====

<img width="809" alt="image" src="https://user-images.githubusercontent.com/33497431/88520692-982cc300-cffc-11ea-9cce-878b17df7a43.png">
768px - do you need so much blank space?

=====

<img width="811" alt="image" src="https://user-images.githubusercontent.com/33497431/88520759-ada1ed00-cffc-11ea-8129-661ae4cf3a5b.png">
white space?

=====

<img width="810" alt="image" src="https://user-images.githubusercontent.com/33497431/88520793-bbf00900-cffc-11ea-9ea8-fd200b43df8c.png">
why do you have much more space between sub-blocks of the same block than between different blocks?

=====

<img width="792" alt="image" src="https://user-images.githubusercontent.com/33497431/88520879-dd50f500-cffc-11ea-9fe0-812c86b9ac58.png">
there is no need to add so much space on mobiles if you have it on desktop design + the image after this block is redundant

=====

<img width="826" alt="image" src="https://user-images.githubusercontent.com/33497431/88520980-ff4a7780-cffc-11ea-8bca-18d0254220a4.png">
something wrong with 'contact us' title

=====

<img width="822" alt="image" src="https://user-images.githubusercontent.com/33497431/88521069-1721fb80-cffd-11ea-86c8-f28a635da152.png">
less indents between title and content, bigger indents between content and edges + add indents between blocks

=====

apply those comments to the smaller screens as well

irenhh commented:
<img width="878" alt="image" src="https://user-images.githubusercontent.com/33497431/88672002-a14a8c80-d0ef-11ea-907b-b53bad4f78e1.png">
the first block still differs from the design (desktop size starts from 992px)

=====

what happens to links and buttons, they are not interactive and there is no hover

=====

<img width="385" alt="image" src="https://user-images.githubusercontent.com/33497431/88672184-d8b93900-d0ef-11ea-8a7d-694133a9d2b3.png">
not fixed

=====

<img width="830" alt="image" src="https://user-images.githubusercontent.com/33497431/88672300-f9818e80-d0ef-11ea-893a-343972d716ba.png">
not fixed

=====

<img width="854" alt="image" src="https://user-images.githubusercontent.com/33497431/88672343-04d4ba00-d0f0-11ea-9efb-e737b95ce353.png">
not fixed

=====

<img width="854" alt="image" src="https://user-images.githubusercontent.com/33497431/88672388-11f1a900-d0f0-11ea-850d-e7f0e1168f5b.png">
not fixed

=====

<img width="849" alt="image" src="https://user-images.githubusercontent.com/33497431/88672420-1d44d480-d0f0-11ea-93c9-c11fa28026a2.png">
not fixed

=====

what is problem with fixing? you didn't follow all comments and mobile design is not ready at all

DmJSCourse commented:
> fix underline, it needs to have the same width as the content + transition is too slow

width is now correct, transition 2 times faster

> indents (check the scree size on the screenshot)

changed

> text should be aligned to the left of the decorative line

it is now aligned

> 1280px is still a desktop size, fix your page for this size

breakpoints are mostly taken from task description and have changes in them compared to default design


> 768px - do you need so much blank space?

it is removed

> white space?

such is the design. In fact, I added it according to previous comments.

> why do you have much more space between sub-blocks of the same block than between different blocks?

to change the design as little as possible. Anyway, it is adjusted and decorative lien added.


> there is no need to add so much space on mobiles if you have it on desktop design + the image after this block is redundant

Agreed. This is fixed.


> something wrong with 'contact us' title

Switched overline to underline.


> less indents between title and content, bigger indents between content and edges + add indents between blocks

Done

irenhh commented:
<img width="163" alt="image" src="https://user-images.githubusercontent.com/33497431/88910654-771adb00-d265-11ea-8807-941387bb43bc.png">
there is nothing in the design about lines here

=====

check your demo link: no link works in the header

=====

<img width="730" alt="image" src="https://user-images.githubusercontent.com/33497431/88910813-adf0f100-d265-11ea-90b5-e326d099d020.png">
this is design, if you want to have such big indents between title and the content, you need to add even bigger indents between title and the edge of the block. It's not very good looking for tablet size, so that I recommend you to reduce indents between title and content
<img width="787" alt="image" src="https://user-images.githubusercontent.com/33497431/88910988-e0025300-d265-11ea-9d15-58080847be78.png">
yours looks like this, chaos with indents

irenhh commented:
header links
